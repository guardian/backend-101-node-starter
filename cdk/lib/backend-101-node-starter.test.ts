import { App } from "aws-cdk-lib";
import { Template } from "aws-cdk-lib/assertions";
import { Backend101NodeStarter } from "./backend-101-node-starter";

describe("The Backend101NodeStarter stack", () => {
  it("matches the snapshot", () => {
    const app = new App();
    const stack = new Backend101NodeStarter(app, "Backend101NodeStarter", { stack: "playground", stage: "TEST" });
    const template = Template.fromStack(stack);
    expect(template.toJSON()).toMatchSnapshot();
  });
});
