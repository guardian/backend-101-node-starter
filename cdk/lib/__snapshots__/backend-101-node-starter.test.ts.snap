// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Backend101NodeStarter stack matches the snapshot 1`] = `
{
  "Metadata": {
    "gu:cdk:constructs": [],
    "gu:cdk:version": "TEST",
  },
}
`;
